htmlRebuild.html

```diff
@@ -0,0 +1,54 @@
+          <li>FORUM</li>
+          <li>ONLINE SHOP</li>
+        </ul>
+      </nav>
+      <section>
+        <h1>OLD SCHOOL PERFECTION // TAKANOBU’S TOYOTA SPRINTER TRUENO.</h1>
+      </section>
+      <article>
+        <p>It’s impossible for you to forget the first car you took for a spin after you got your driver’s license. Windows down, your favorite music on the radio, and the feeling of freedom on the open road. Whether it was the parent’s mini-van, or your $400 sedan—you will always hold a special place in your heart for your first cruiser. For our friend, Takanobu Sugahara from Hiroshima, Japan, his first licensed joy-ride was in his Toyota AE86. After getting rid of his first, he missed it dearly. He promised himself that he would own another AE86 again when he found the right one.</p>
+        <img></img>
+        <img></img>
+        <p>Some six months ago, fate brought Takanobu-san together with an immaculate stock 1985 Sprinter Trueno Apex 2CP. Takanobu-san wanted to build an AE86 that was quick, looked great, and was reliable enough to daily. Years of build plans came to fruition soon after he had the car in his possession.</p>
+        <img></img>
+        <img></img>
+        <p>One of Takanobu-san’s favorite things about the car is the engine. He said from the exterior his car doesn’t look quick, “but it’s fast!” The car has a more powerful “red-top” 4A-GE power-plant from the Toyota AE92. He also ported the motor, added a metal head-gasket, big valve kit, Escort 82.5 cast pistons, Toyota Vitz lifters, Toyota AE111 connecting rod, individual throttle bodies, TODA Racing high camshafts (IN 304, EX 288), TODA Racing adjustable camshaft pulley, and a Freedom computer to tune it all properly.</p>
+        <img></img>
+        <p>Takanobu-san is also running a TODA Racing lightweight flywheel, SECTION racing clutch, and a TRD 2-way differential. So, not only does the car look great, it’s also fairly fast!</p>
+        <img></img>
+        <p>The biggest challenge Takanobu-san had with his AE86 was dialing in the fitment and cutting the stock body to add his other favorite part of the car, the Crystal Body Yokoyama (CBY) over-fenders. The task of cutting the body wasn’t difficult; it was bringing himself to cut the car. After that was done, he had a hard time finding the right tire size. He says, “They didn’t fit and so I have bought brand new sets of tires 3 times. Even then, the tire still rubs, so I’m currently working on sorting out the front suspension.”</p>
+        <img></img>
+        <img></img>
+      </article>
+      <section>
+        <h2>1985 Sprinter Trueno Apex 2CP</h2>
+        <ul>
+          <li>Bodykit</li>
+          <li>Wheels</li>
+          <li>Suspension</li>
+          <li>Engine</li>
+          <li>Drive line</li>
+          <li>Interior</li>
+        </ul>
+      </section>
+      <footer>
+        <h1>RELATED POSTS</h1>
+      </footer>
+    </header>
+  </body>
+</html>
```

Comment:
Add this to the end of your user preferences:

`"ensure_newline_at_eof_on_save": true`